### DISCUSSION TITLE
Can Github organization repos be imported?

### DISCUSSION THREAD

**Discussion by metamn** [vercel/community#123] [Help] [answered]
I'm a Github @user owning an @organization which has a project I want to deploy. However the import dialog lets me select only @user repos and no @organization repos.

![image](https://user-images.githubusercontent.com/86295/82287150-34589f00-99a8-11ea-83d3-0dc5da022f4b.png)

**Comment by mcsdevv** [comment]
Hello @metamn, GitHub organizations can only be connected to Vercel team accounts. Are you trying to import the project to your Hobby account?

**Reply by cntran** [reply]
How do you connect a github organization to a team account? I can't find anywhere how to do this?

**Comment by metamn** [comment]
Yes

On Tue, May 19, 2020 at 15:12 Matthew Sweeney <notifications@github.com>
wrote:

> Hello @metamn <https://github.com/metamn>, GitHub organizations can only
> be connected to Vercel team accounts. Are you trying to import the project
> to your Hobby account?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zeit/now/discussions/4413#discussioncomment-16188>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAVCF54SSMHVUKB2CVXVT3RSJZSBANCNFSM4NEVIS7Q>
> .
>
-- 
bartus.csongor@gmail.com
http://metamn.io

**Comment by mcsdevv** [comment, marked as answer]
That'll be why then, Vercel Hobby accounts can only be connected to GitHub personal accounts. To connect a GitHub organization to Vercel, you will need to create a Vercel team first of all.

**Reply by sealsrock12** [reply]
Why though? Teams seem to cost money, and can't be used with a subdomain of a personal account.

**Reply by flawnn** [reply]
That's my problem right now

**Comment by metamn** [comment]
Thanks Matthew !!!

On Tue, May 19, 2020 at 18:06 Matthew Sweeney <notifications@github.com>
wrote:

> That'll be why then, Vercel Hobby accounts can only be connected to GitHub
> personal accounts. To connect a GitHub organization to Vercel, you will
> need to create a Vercel team first of all.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zeit/now/discussions/4413#discussioncomment-16248>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAVCF6DGYDNNRBCGUWZ3YTRSKN6JANCNFSM4NEVIS7Q>
> .
>
-- 
bartus.csongor@gmail.com
http://metamn.io

**Comment by paulogdm** [comment]
@metamn Thanks for your feedback. We have relayed it to the team.

If you have any questions, we are here.

**Comment by Pieparker** [comment]
This topic is rather old now and has been marked as Answered. If you’re exhibiting the same or a similar issue to the original poster, please feel free to create your own discussion and provide as much context as you can. This helps us to ensure that discussions are relevant, fresh and free of any possibly misleading context.
